OrbitQt/orbitmainwindow.cpp

```diff
@@ -353,7 +353,7 @@ void OrbitMainWindow::on_actionReport_Bug_triggered() {
 
 void OrbitMainWindow::on_actionAbout_triggered() {
   OrbitQt::OrbitAboutDialog dialog{this};
-  dialog.setWindowTitle(windowTitle());
+  dialog.setWindowTitle("About Orbit");
```

Comment:
Do you want the "Orbit" part here when this will be followed by "Orbit Profiler ..." (maybe yes, just pointing this out)?

Reply:
I'm undecided here. What do you prefer? `About - Orbit Profiler 1.52 [BETA]` or `About Orbit - Orbit Profiler 1.52 [BETA]`?

Reply:
I think I prefer the first option. Not that critical though obviously.

Reply:
Great. Let's take the first one. Absolutely fine with me! Thanks!